generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.ts.ejs

```diff
@@ -81,7 +82,11 @@ export class AlertService {
 
     <%_ if (enableTranslation) { _%>
     if (alert.translationKey) {
-      alert.message = this.translateService.instant(alert.translationKey, alert.translationParams);
+      const translatedMessage = this.translateService.instant(alert.translationKey, alert.translationParams);
+      // if translation key exists
+      if (translatedMessage !== `${translationNotFoundMessage}[${alert.translationKey}]`) {
```

Comment:
```suggestion
      if (!alert.message || translatedMessage !== `${translationNotFoundMessage}[${alert.translationKey}]`) {
```
Use translatedMessage if no message.

Can you add tests for alerts having translationKey at
https://github.com/jhipster/generator-jhipster/blob/22adabd1f9bf4c9562f723fb91045055d67123aa/generators/client/templates/angular/src/main/webapp/app/core/util/alert.service.spec.ts.ejs?